4-database/1-pagila/challenge.sql

```diff
@@ -0,0 +1,104 @@
+
+/*
+    Challenge 1.
+    Write a SQL query that counts the number of films in each category in the Pagila database.
+    - The query should return two columns: category and film_count
+    - category should display the name of each category
+    - film_count should show the total number of films in that category
+    - Results should be grouped by category name
+ */
+
+
+-- your query here
+
+
+ /*
+    Challenge 2.
+    Write a SQL query that finds the top 5 customers who have spent the most money in the Pagila database.
+    - The query should return three columns: first_name, last_name, and total_spent
+    - first_name and last_name should display the customer's full name
```

Comment:
WIth this item i'm not understanding if I need to return the 3 columns as you say on first bullet point or should i do a concat and return firstname and lastname as fullname in which case the result is just 2 columns